packages/vm/src/params.ts

```diff
@@ -78,7 +78,7 @@ export const paramsVM: ParamsDict = {
   7002: {
     // config
     systemAddress: '0xfffffffffffffffffffffffffffffffffffffffe', // The system address to perform operations on the withdrawal requests predeploy address
-    withdrawalRequestPredeployAddress: '0x00A3ca265EBcb825B45F985A16CEFB49958cE017', // Address of the validator excess address
+    withdrawalRequestPredeployAddress: '0x05F27129610CB42103b665629CB5c8C00296AaAa', // Address of the validator excess address
```

Comment:
This is the address of this commit: https://github.com/ethereum/EIPs/commit/133576571773222705b93372fc9cad7f2a17f39e

The latest commit (which shows up on eips.ethereum.org) is https://github.com/ethereum/EIPs/commit/a23be81719b713f5e59b5e040e03ad88c0fb2e3d pointing to `0x0511Ce19514e1298Fba96de582652A016E2CAaAa`

Reply:
yes @parithosh mentioned there are a few more updates coming through, so we gotta hold on